ComSemApp/teacher/urls.py

```diff
@@ -24,4 +24,10 @@
 
     url(r'^corpus/search$', corpus_views.corpus_search, name='corpus_search'),
     url(r'^corpus/error_search', corpus_views.error_search, name='error_search'),
+
+
+    url(r'^course/(?P<course_id>[0-9]+)/requests/$', views.SpeakingPracticeReviewRequestsListView.as_view(), name='request_list'),
```

Comment:
hmmm... we might want to consider adding "speakingpractice" into the urls after course number